### DISCUSSION TITLE
SSE Server-sent event for records API

### DISCUSSION THREAD

**Discussion by gianluca-sabena** [strimzi/strimzi-kafka-bridge#862] [Q&A]
I looked at record API but I haven't found any reference to SSE server-sent event.

Is it possible to subscribe to /records and get a stream of record pushed from the server to the http client as soon as they arrive via SSE server-sent event?

Thank you

**Comment by scholzj** [comment]
No, this is currently not supported. You have to poll for the new records.

**Reply by gianluca-sabena** [reply]
Thank you. Is this something you may evaluate in the future releases?

**Reply by ppatierno** [reply]
I don't think so. We don't have any plan to support like this.
